### PR TITLE
Implement assign permission to role feature

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i write assign permission to role feature according to intermediary model
#30 
image
image

### CODE REVIEW

File: myproject/supermarket_manager/models.py [hunk 1]
```diff
@@ -75,6 +75,16 @@ class Permission(models.Model):
     permission_name = models.CharField(max_length=100, unique=True)
     description = models.CharField(max_length=100)
     status = models.CharField(max_length=1, choices=STATUS_CHOICES, default="A")
+    
+    def __str__(self):
+        """
+        Return a string representation of the Role object.
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/models.py:81]: Please double check this docstring.

File: myproject/supermarket_manager/views.py [hunk 2]
```diff
@@ -1,14 +1,15 @@
 from django.http import JsonResponse
 from django.db.utils import IntegrityError
 from django.views.decorators.csrf import csrf_exempt
-from .models import Role, Permission
+from .models import Role, Permission, Role_Permission
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/views.py:4]: Please import with the format: from .models import ( .... )

File: myproject/supermarket_manager/urls.py [hunk 3]
```diff
@@ -18,4 +18,5 @@
     path("api/permissions/adds", add_permission, name="add-permission"),
     path('api/permissions/get_permission', get_permissions, name='get-permission'),
     path("api/permissions/updates/<str:permission_id>", update_permission, name="update-permission"),
+    path("api/role-permissions/assigns", assign_permission, name="assign-permission")
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/urls.py:21]: Please substitute a more appropriate endpoint.

File: myproject/supermarket_manager/models.py (hunk 1 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/models.py]: Done

File: myproject/supermarket_manager/views.py (hunk 2 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

File: myproject/supermarket_manager/urls.py (hunk 3 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/urls.py]: Done

**Pull request merged**